libs/blocks/global-navigation/global-navigation.js

```diff
@@ -512,9 +512,20 @@ class Gnav {
       return 'linux';
     };
 
+    const scriptUrl = `https://${environment}.adobeccstatic.com/unav/1.1/UniversalNav.js`;
+    const styleUrl = `https://${environment}.adobeccstatic.com/unav/1.1/UniversalNav.css`;
+
     await Promise.all([
-      loadScript(`https://${environment}.adobeccstatic.com/unav/1.1/UniversalNav.js`),
-      loadStyle(`https://${environment}.adobeccstatic.com/unav/1.1/UniversalNav.css`),
+      loadScript(scriptUrl),
+      loadStyle(styleUrl, (e) => {
```

Comment:
Would the error be part of the callback and not simply propagate? 🤔 
Why did you add the lana log here, but nowhere else for loadStyle/loadScript in this case?

Reply:
This code is part of `decorateUniversalNav` and is again used in `imsReady` as task with the existing lana logs as showd in previous comment. While an error for loadScript is already captured [here](https://github.com/adobecom/milo/blob/e6731831f7ecf2ceff64eb66b59a001f7c12b110/libs/utils/utils.js#L404), there isn't one for loadStyle. Currently, if there's a network issue while loading the CSS, the universalNav will display incorrectly like this:
<img width="228" alt="Screenshot 2024-06-03 at 4 19 20 PM" src="https://github.com/adobecom/milo/assets/41122199/5a788512-6966-4d26-a7be-f54b27d76ae4">

Which is leading to a poor user experience. Therefore, the existing LANA log in `imsReady` doesn't capture this, which is why the log is now added for loadStyle.

@mokimo do you think this should be covered [here](https://github.com/adobecom/milo/blob/e6731831f7ecf2ceff64eb66b59a001f7c12b110/libs/utils/utils.js#L317) by thorwing an error?

Reply:
I don't see a lot of value adding this to the utils unless there is a good use-case for tracking this and have people act based on the logs.

If this you see this enough in the gnav, having a gnav-util for this would be more appropriate. If you want to track when/how this happens just in the universalNav case, this is fine 👍

Reply:
Instead of `loadStyle` from `utils`, I have used `loadStyles` from `utilities` of global navigation.